### PR TITLE
Bayc tutorial

### PR DESCRIPTION
**Pull request by abernatskiy**
A sketch of the first two parts.

### CODE REVIEW

File: docs/tutorials/bayc/step-one-indexing-transfers.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Step 1: Generating a starter"
+description: >-
+  Autogenerated squid indexing Transfer events
+sidebar_position: 10
+---
+
+# Step 1: Generating a starter
+
+In the course of this tutorial we will build a squid that gets data about [Bored Ape Yacht Club](https://boredapeyachtclub.com) NFTs, their transfers and owners from the [Ethereum blockchain](https://ethereum.org) and [IPFS](https://ipfs.tech/), stores it in a database and serves it over a GraphQL API. Here we do the first step: generate a squid that indexes just the `Transfer` events emitted by the [BAYC token contract](https://etherscan.io/address/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d) and remove the unneeded parts.
```
Comment (dzhelezov) [inline docs/tutorials/bayc/step-one-indexing-transfers.md:10]: which data? should be more precise

File: docs/tutorials/bayc/step-one-indexing-transfers.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Pre-requisites: Node.js, [Subsquid CLI](/squid-cli/installation), Docker. 
+
+## Generating a squid
```
Comment (dzhelezov) [inline docs/tutorials/bayc/step-one-indexing-transfers.md:14]: Need a better title

File: docs/tutorials/bayc/step-two-deriving-owners-and-tokens.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+import assert from 'assert'
+
+interface Entity {
+    id: string
+    constructor: {
+        name: string
+    }
+}
+
+export class EntityGenerator {
```
Comment (dzhelezov) [inline docs/tutorials/bayc/step-two-deriving-owners-and-tokens.md:121]: So why exactly do we need EntityGenerator?

**Review by dzhelezov — CHANGES_REQUESTED** [review]

File: docs/tutorials/bayc/step-two-deriving-owners-and-tokens.md (hunk 3 above)
Comment (belopash) [reply docs/tutorials/bayc/step-two-deriving-owners-and-tokens.md:121]: Looks too complex for such simple project

File: docs/tutorials/bayc/step-one-indexing-transfers.md (hunk 1 above)
Comment (abernatskiy) [reply docs/tutorials/bayc/step-one-indexing-transfers.md:10]: Now that I've kept this paragraph in the new version, I must ask you - why do we need a clarification here? I presume that the reader would not be interested in the exact data we get here, but in which _kinds_ of data we could've retrieved in principle. The sentence gives a nice, short overview of that, no?

**Pull request closed without merging**